Python/Tests/DebuggerTests/DebuggerTests.cs

```diff
@@ -1952,10 +1952,12 @@ private enum ExceptionMode {
                     PythonThread bpThread = null;
                     bool wrongLine = false;
                     proc.BreakpointHit += (sender, args) => {
+                        var continueEvent = new AutoResetEvent(true);

```

Comment:
As far as scoping goes, I had to move `Continue` out of the callback anyway (see my reply to Rick below). But it can still have a `using` - and I'll add the same for other events in this test. I'll make a separate change to fix other tests to avoid spamming this CR.